joern-cli/frontends/pysrc2cpg/src/test/scala/io/joern/pysrc2cpg/dataflow/DataFlowTests.scala

```diff
@@ -589,4 +591,25 @@ class RegexDefinedFlowsDataFlowTests
     flows.size shouldBe 2
   }
 
+  "flow across interprocedural module variables" in {
+    val cpg: Cpg = code(
+      """
+        |a = 42
+        |""".stripMargin,
+      "foo.py"
+    )
+      .moreCode(
+        """
+          |import foo
+          |
+          |print(foo.a)
+          |""".stripMargin,
+        "bar.py"
+      )
+
+    val source = cpg.literal("42").l
+    val sink   = cpg.call.code("print.*").argument.l
+    sink.reachableByFlows(source).size shouldBe 1
```

Comment:
Because of this PR, we can now obtain the following flow:
```
    /*
    ____________________________________________________________
    | nodeType| tracked         | lineNumber| method   | file   |
    |===========================================================|
    | Literal | <module>.a = 42 | 2         | <module> | foo.py |
    | Call    | print(foo.a)    | 4         | <module> | bar.py |
     */
```